### PR TITLE
Make main v2 free

### PR DESCRIPTION
**Pull request by b-onc**
Tried to keep commits as separate as possible...

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,829 +6,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 # Upcoming
 
-### 🔄 Changed
-
```
Comment (VojtaStavik) [inline CHANGELOG.md:10]: I don't think we should remove the old stuff! Just add a new version :)

**Review by VojtaStavik — APPROVED** [review]
🚢 🇮🇹

File: CHANGELOG.md (hunk 1 above)
Comment (b-onc) [reply CHANGELOG.md:10]: it'll be interesting then, old CHANGELOG will continue (2.6.4, 2.7, 2.7.1....) but this changelog will jump from 2.6.2 -> 3.0.0

Comment (VojtaStavik) [reply CHANGELOG.md:10]: All right with me 👍

**Pull request merged**